src/med/display.d

```diff
@@ -593,12 +593,12 @@ Lout:
 			}
 			if (inmark == 2)
 			    inmark = 0;
+			if (!nextLine)
+			    break;
```

Comment:
Why? It's broken on Windows.

Reply:
Weird, I can't remember typing in that ... No idea how that sneaked in! :)